### PR TITLE
Find lang.sh multi names

### PR DESCRIPTION
**Pull request by proyvind**

### CODE REVIEW

File: scripts/find-lang.sh [hunk 1]
```diff
@@ -54,7 +55,7 @@ fi
 shift
 
 if [ -z "$1" ] ; then usage
-else NAME=$1
+else NAMES[0]=$1
```
Comment (ignatenkobrain) [inline scripts/find-lang.sh]: IIRC it is bashism, arrays are not supported in POSIX shell

Comment (Conan-Kudo) [inline scripts/find-lang.sh]: Arrays are supported in POSIX shell, but not like that. They're way more rudimentary.

Comment (ignatenkobrain) [inline scripts/find-lang.sh]:
```
$ dash -c "NAMES[0]='x'"
dash: 1: NAMES[0]=x: not found
```

**Comment by proyvind** [comment]
I went with bash as interpreter;)

File: scripts/find-lang.sh [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
 # 2011-11-16 Per Øyvind Karlsen <peroyvind@mandriva.org>
 #   * add support for HTML files (from Mandriva)
+#   * add support for multiple names
```
Comment (Conan-Kudo) [inline scripts/find-lang.sh]: This distinctly didn't happen in 2011.

Comment (proyvind) [reply scripts/find-lang.sh]: well, the changelog entry and the original commit (@cooker) did take place in 2011, but sure, the commit to rpm.org repo didn't..
Same for the previous ("add support for HTML files (from Mandriva)" that's already merged in master) too btw.
I'm occupied with other stuff currently, so don't expect any nitpicking commits from my end anytime soon, but please don't let me stop others from taking responsibility on meanwhile. ;)

**Comment by pmatilai** [comment]
Changelogs embedded in individual files don't belong to a project maintained in a VCS to begin with.

**Comment by pmatilai** [comment]
Merged the authors from the find-lang.sh "changelog" into CREDITS where they belong, and cleaned up the commit message and merged as a single commit 2249cf4507545a1f73c52941b32d063ba72edb35.

**Comment by pmatilai** [comment]
Oh and thanks for the patch, I do remember cursing at the stupid limitation on a few occasions :)

**Pull request closed without merging**